From: Maximilian Weigand <mweigand@mweigand.net>
Date: Tue, 1 Aug 2023 21:36:02 +0200
Subject: backlight: lm3630a: fix dts error checking

---
 drivers/video/backlight/lm3630a_bl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/backlight/lm3630a_bl.c b/drivers/video/backlight/lm3630a_bl.c
index 6393647..1eecb0c 100644
--- a/drivers/video/backlight/lm3630a_bl.c
+++ b/drivers/video/backlight/lm3630a_bl.c
@@ -501,7 +501,7 @@ static int lm3630a_parse_bank(struct lm3630a_platform_data *pdata,
 			break;
 		}
 	}
-	if (!current_index){
+	if (current_index == -1){
 		return -EINVAL;
 	}
 	if (!ret) {
