From: Maximilian Weigand <mweigand@mweigand.net>
Date: Sat, 27 Jul 2024 03:34:27 +0800
Subject: rockchip-ebc: add note for potential future improvements

---
 drivers/gpu/drm/rockchip/rockchip_ebc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/rockchip/rockchip_ebc.c b/drivers/gpu/drm/rockchip/rockchip_ebc.c
index 95adfc1..734d920 100644
--- a/drivers/gpu/drm/rockchip/rockchip_ebc.c
+++ b/drivers/gpu/drm/rockchip/rockchip_ebc.c
@@ -2379,6 +2379,8 @@ static int rockchip_ebc_drm_init(struct rockchip_ebc *ebc)
 		return ret;
 
 	ebc->encoder.possible_crtcs = drm_crtc_mask(&ebc->crtc);
+	// todo: consider using drmm_simple_encoder_alloc()
+	// see: https://www.kernel.org/doc/html/latest/gpu/drm-kms-helpers.html?highlight=drm_simple_encoder_init#c.drm_simple_encoder_init
 	ret = drm_simple_encoder_init(drm, &ebc->encoder, DRM_MODE_ENCODER_NONE);
 	if (ret)
 		return ret;
